DocSum/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -21,35 +21,29 @@ This section describes how to quickly deploy and test the DocSum service manuall
 6. [Test the Pipeline](#test-the-pipeline)
 7. [Cleanup the Deployment](#cleanup-the-deployment)
 
-### Access the Code
+### Access the Code and Set Up Environment
 
 Clone the GenAIExample repository and access the ChatQnA Intel Xeon platform Docker Compose files and supporting scripts:
 
-```
+```bash
 git clone https://github.com/opea-project/GenAIExamples.git
-cd GenAIExamples/DocSum/docker_compose/intel/cpu/xeon/
+cd GenAIExamples/DocSum/docker_compose
+source set_env.sh
+cd intel/cpu/xeon/
 ```
 
-Checkout a released version, such as v1.2:
+NOTE: vLLM is set to execute warmup by default. If you'd like to skip warmup, `export VLLM_SKIP_WARMUP=true`
```

Comment:
If you want to give more details, text here (and below) could be expanded to something like this:
```suggestion
NOTE: by default vLLM does "warmup" at start, to optimize its performance for the specified model and the underlying  platform, which can take long time. For development (and e.g. autoscaling) it can be skipped with `export VLLM_SKIP_WARMUP=true`.
```